### DISCUSSION TITLE
Place Lanterns on walls

### DISCUSSION THREAD

**Discussion by Pilzinsel64** [Roadhog360/Et-Futurum-Requiem#413] [Improvements]
Looks like Lanterns on 1.19.2 can be placed on walls. Not sure if this is easy to backport but it would be a nice improvement! 🙂

![grafik](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/23138465/390a4431-6203-4080-96da-c264a13c6b49)

Thank you for your hard, I really love this mod!

**Comment by Roadhog360** [comment]
Not vanilla

**Reply by Pilzinsel64** [reply]
Oh sorry, missed that. I normally can keep the overview but sometimes sadly I still miss something.
